### PR TITLE
feat(uk): Introduce Ukrainian language

### PR DESCRIPTION
**Pull request by podarok**
Ready for review

### CODE REVIEW

**Comment by podarok** [comment]
Please, review @damianopetrungaro )))

**Comment by damianopetrungaro** [comment]
Noice!
Thanks @podarok !

**Comment by podarok** [comment]
@damianopetrungaro done. Please check

File: content/v1.0.0/index.uk.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+2. **feat:** коміт _типу_ `feat` додає новий функціонал до коду (це корелюється з [`MINOR`](https://semver.org/lang/uk/) в Семантичних Версіях).
+3. **BREAKING CHANGE:** коміт, що має додаток `BREAKING CHANGE:`, або суфікс `!` після типу, додає зміну в API (це корелюється з [`MAJOR`](https://semver.org/lang/uk/) в Семантичних Версіях).
+BREAKING CHANGE може бути частиною будь якого _type_.
+4. _types_ інші ніж `fix:` та `feat:` дозволені, для прикладу [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional) (базовано на [Angular домовленостях](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines)) рекомендовані `build:`, `chore:`,
+  `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, та інші.
+5. _footers_ інші ніж `BREAKING CHANGE: <опис>` можуть бути теж і наслідувати угоду схожу до
+  [git trailer format](https://git-scm.com/docs/git-interpret-trailers).
+
+Додаткові типи не заборонені угодою про Політику Комітів, і не накладають додаткові умови в Семантичних Версія ( окрім випадків коли вони містять BREAKING CHANGE).
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:43]: Прибрати пробіл після дужки (

File: content/v1.0.0/index.uk.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+---
+
+<br />
+Коміт містить наступні елементи структури, для спілкування інтенціх для споживачів
+вашої бібліотеки:
+
+1. **fix:** коміт _типу_ `fix` виправляє ваду в вашому коді (це корелюється з [`PATCH`](https://semver.org/lang/uk/) в Семантичних Версіях).
+2. **feat:** коміт _типу_ `feat` додає новий функціонал до коду (це корелюється з [`MINOR`](https://semver.org/lang/uk/) в Семантичних Версіях).
+3. **BREAKING CHANGE:** коміт, що має додаток `BREAKING CHANGE:`, або суфікс `!` після типу, додає зміну в API (це корелюється з [`MAJOR`](https://semver.org/lang/uk/) в Семантичних Версіях).
+BREAKING CHANGE може бути частиною будь якого _type_.
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:37]: будь якого пишеться з дефісом.

File: content/v1.0.0/index.uk.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+
+1. Коміти MUST(повинні) бути з префіксом типу, що складається з іменника `feat`, `fix`, тощо., наступним йде
+OPTIONAL(необов'язково) область(сфера), OPTIONAL(необов'язково) `!`, та REQUIRED(обов'язково) завершити двокрапкою та пропуском.
+2. Тип `feat` MUST(повинен) використовуватися, коли коміт додає новий функціонал в ваш додаток або бібліотеку.
+3. Тип `fix` MUST(повинен) використовуватися якщо коміт є латкою для вашого додатку.
+4. Область(сфера) MAY(може) бути вказана після типу. Область(сфера) MUST(повинна) містити іменник, що описує секцію коду виділену лапками, наприклад, `fix(parser):`
+5. Опис MUST(повинен) одразу завершатися двокрапкою і пропуском після області(сфери).
+Опис - це короткий підсумок змін в коді, наприклад, _fix: array parsing issue when multiple spaces were contained in string_.
+6. Довге тіло коміту MAY(може) бути додано після короткого опису, надаючи додаткову контекстру інформацію про зміни в коді. Тіло MUST(повинне) починатися з однієї пустої строки після опису.
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:101]: Неправильне слово "контекстру"

File: content/v1.0.0/index.uk.md [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+7. Тіло коміту - це вільна форма і може містити будь яку кількість нових рядків і параграфів.
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:102]: будь яку через дефіс

Comment (anprok) [reply content/v1.0.0/index.uk.md:102]: "параграфів" замінити на "абзаців"

File: content/v1.0.0/index.uk.md [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+10. Значення додатку MAY(може) містити пропуски та нові рядки, і зчитування MUST(повинне) припинятися коли наступний коректний токен/роздільник додатку знайдено.
+11. BREAKING CHANGE MUST(повинні) бути виділені в типі/області префіксі коміту, або як вміст додатку.
+12. Якщо виділено в додатку, BREAKING CHANGE MUST(повинно) містити великими літерами BREAKING CHANGE, потім двокрапка, пропуск, та опис, наприклад,
+_BREAKING CHANGE: environment variables now take precedence over config files_.
+14. Якщо вставлене в типі/області префіксом, BREAKING CHANGE MUST(повинно) бути виділено `!` одразу перед `:`. Якщо `!` використане, `BREAKING CHANGE:` MAY(може) бути упущене в додатку, і опис коміту SHALL(має) бути використаний для опису BREAKING CHANGE.
+15. Типи, інші за `feat` та `fix` MAY(можуть) бути використані в ваших коміт повідомленнях, наприклад, _docs: updated ref docs._
+16. Одиниці інформаціх, що роблять Політику Комітів MUST NOT(не повинні) бути сприйняті як чутливими до регістру творцями інструментів, за виключенням BREAKING CHANGE, що MUST(мусить) бути великими літерами.
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:112]: інформаціх - інформації

File: content/v1.0.0/index.uk.md [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+* Автоматичне визначення наступної версії в Семантичних Версіях(базуючись на типах комітів, що увійшли).
+* Комунікування природи змін для команди, в публічному просторі, до власників.
+* Запуск процесів компіляції та публіації.
+* Полегшує людям шлях до контрибуції в ваш проєкт, надаючи їм більш структуровану історію комітів.
+
+## ЧЗП
+
+### Як потрібно поводити себе на початковій фазі розробки?
+
+Рекомендуємо продовжувати так, наче ви вже створили реліз продукту. Типічно *хтось*, навіть, якщо це ваші розробники, вже використовують ваше програмне забезпечення. Вони будуть знати, що виправлено, що поламано тощо.
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:127]: Типічно - типово

File: content/v1.0.0/index.uk.md [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+
+### Як поводити себе з великими і малими літерами?
+
+Будь які можуть використовуватися, вле варто притримуватися якогось одного підходу.
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:131]: Будь які через дефіс

File: content/v1.0.0/index.uk.md [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+
+### Що робити, якщо коміт підходить для кількох типів одночасно?
+
+Зробіть декілька комітів, наскільки це можливо. Частина вигоди від Політики Комітів є можливіть спонукати нас робити більш організовані коміти і PR/MRs
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:135]: можливіть - можливість

File: content/v1.0.0/index.uk.md [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+<тип>[додатково]: <опис>
+
+[необов'язково тіло]
+
+[необов'язково додаток(тки)]
+```
+---
+
+<br />
+Коміт містить наступні елементи структури, для спілкування інтенціх для споживачів
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:31]: інтенціх?

File: content/v1.0.0/index.uk.md [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+15. Типи, інші за `feat` та `fix` MAY(можуть) бути використані в ваших коміт повідомленнях, наприклад, _docs: updated ref docs._
+16. Одиниці інформаціх, що роблять Політику Комітів MUST NOT(не повинні) бути сприйняті як чутливими до регістру творцями інструментів, за виключенням BREAKING CHANGE, що MUST(мусить) бути великими літерами.
+17. BREAKING-CHANGE MUST(повинно) бути синонімом до BREAKING CHANGE, коли використовується в додатку.
+
+## Для чого використовувати Політику Комітів.
+
+* Автоматичне створення Нотаток про Реліз(Changelog)
+* Автоматичне визначення наступної версії в Семантичних Версіях(базуючись на типах комітів, що увійшли).
+* Комунікування природи змін для команди, в публічному просторі, до власників.
+* Запуск процесів компіляції та публіації.
```
Comment (anprok) [inline content/v1.0.0/index.uk.md:120]: публікації

**Review by anprok — CHANGES_REQUESTED** [review]

File: content/v1.0.0/index.uk.md (hunk 10 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

File: content/v1.0.0/index.uk.md (hunk 9 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

File: content/v1.0.0/index.uk.md (hunk 8 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

File: content/v1.0.0/index.uk.md (hunk 7 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

File: content/v1.0.0/index.uk.md (hunk 6 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

File: content/v1.0.0/index.uk.md (hunk 5 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

File: content/v1.0.0/index.uk.md (hunk 4 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

File: content/v1.0.0/index.uk.md (hunk 3 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

File: content/v1.0.0/index.uk.md (hunk 2 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

File: content/v1.0.0/index.uk.md (hunk 1 above)
Comment (podarok) [reply content/v1.0.0/index.uk.md]: solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

**Comment by podarok** [comment]
Thanks for the review @anprok 
solved https://github.com/conventional-commits/conventionalcommits.org/pull/376/commits/9f91fe0ec5e183296c52da31b2a2d722a909a07a

@damianopetrungaro I guess it is ready now

**Comment by damianopetrungaro** [comment]
Thanks for the hard work @podarok !
Merged :)

**Comment by podarok** [comment]
Thank you @damianopetrungaro 
It's already on a site, confirmed all good

**Pull request merged**